### ISSUE TITLE
Old devices

### ISSUE THREAD

**Issue by p1nox** [floatinghotpot/cordova-plugin-sms#5]
@floatinghotpot have you tried the example in android 4.0.3 or under? I think there is a problem on those. 

I tried in an HTC One S (WebKit/534.30), and see no error while the app it's bootstrapping, even I can see the message that it's on "watching" mode, but when an SMS arrive there is no confirmation, it's like no message had arrived at all.

**Comment by floatinghotpot** [comment]
I tested the plugin on my samsung s4 (android 4.3) and it works pretty well, but not sure any other older versions.

**Comment by p1nox** [comment]
I found the problem. I was using [Handcent SMS](https://play.google.com/store/apps/details?id=com.handcent.nextsms) in this phone, and I think for some reason it was overwriting/blocking our watch listener, I uninstalled the app and it worked :+1:

**Comment by floatinghotpot** [comment]
Good :+1

**Comment by p1nox** [comment]
I'm having this problem again (but on a Nexus 5 with Android 6.0.1), and it might be related to the same reason, also see https://github.com/floatinghotpot/cordova-plugin-sms/issues/23. I already removed the app I was using, [Messenger](https://play.google.com/store/apps/details?id=com.google.android.apps.messaging) which is the recommended sms app, so now is using Hangouts as SMS app. Despite that, `onSMSArrive` event is never fired.

There is a way to set a higher priority for the SMS listener, that would be better than been forced to remove an app from the phone.

According to [this](http://stackoverflow.com/questions/14395925/broadcast-receiver-not-processing-smss), one have to modify an IntentFilter, like here: https://github.com/floatinghotpot/cordova-plugin-sms/blob/master/src/android/SMSPlugin.java#L363 . I already did it, checking with values `999` and `2147483647`, but it doesn't work.

Also I don't know if this is needed as well, but maybe could fixed the problem modifying the manifest using:

```
<intent-filter android:priority="999">
```

or 

```
<intent-filter android:priority="2147483647">
```

Any ideas @floatinghotpot @rgdamalerio ?
